assets/src/activation-notice/app/components/dismiss.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+
+/**
+ * Renders a Dismiss button as required/used by WordPress.
+ *
+ * Does not actually *render* the button itself, but emits an event so that
+ * WordPress detects the notice and registers a click handler.
+ *
+ * @return {*} Rendered component.
+ */
+function Dismiss() {
+  useEffect(() => {
+    document.dispatchEvent(new Event('wp-updates-notice-added'));
```

Comment:
Hacky, but works to retain default WordPress behavior, which is to dismiss the notice with a small fade/slide effect.

Reply:
If it works. 👍